### PR TITLE
[main] EOS-14306 CSM updates "pre-flight" checks

### PR DESCRIPTION
**Pull request by navalkp**
# Backend

## Problem Statement
<pre>
Story Ref (if any):https://jts.seagate.com/browse/EOS-14306
</pre>
## Unit testing on RPM done
<pre>
No
</pre>
## Problem Description
<pre>
Create REST API to check system status
</pre>
## Solution
<pre>
Created 2 REST API to check system status , current check status of consul and elasticsearch
GET system/status
and 

Status call for specific db
GET system/status/<db_name>    <db_name> consul , es
</pre>
## Unit Test Cases
<pre>
Tested with POSTMAN and UI tested pointing to my dev VM
</pre>
Signed-off-by:

### CODE REVIEW

File: csm/core/services/system_status.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    """
+    Provides system status services
+    """
+
+    def __init__(self):
+        super(SystemStatusService, self).__init__()
+        self._action_map = {const.SYSTEM_STATUS_CONSUL: self._get_consul_status,
+            const.SYSTEM_STATUS_ELASTICSEARCH: self._get_elasticsearch_status}
+
+    async def check_status(self, check_list):
```
Comment (prathameshrodi) [inline csm/core/services/system_status.py:34]: We could update `check_list` -> `resources`.

File: csm/core/services/system_status.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        """
+        Validate status of resource
+        :param check_list: list of resources.
+        :return:
+        """
+
+        resp = dict()
+        resp[const.SYSTEM_STATUS_SUCCESS] = True
+        for each_resource in check_list:
+            Log.debug(f" each resource: {each_resource}")
```
Comment (prathameshrodi) [inline csm/core/services/system_status.py:44]: `each resource` doesn't define enough about the message.
Please update it to a little descriptive about the values expected here.

**Review by prathameshrodi — CHANGES_REQUESTED** [review]

File: csm/core/services/system_status.py (hunk 1 above)
Comment (navalkp) [reply csm/core/services/system_status.py]: changed to resources

File: csm/core/services/system_status.py (hunk 2 above)
Comment (navalkp) [reply csm/core/services/system_status.py]: Updated message

**Review by prathameshrodi — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**